tests/test_cli.py

```diff
@@ -868,3 +868,19 @@ def test_invoke_invalid_command(runner, tmpdir):
     assert (
         'Error: Invalid setting for [main][default_command]' in result.output
     )
+
+
+def test_show_priority(runner, todo_factory, todos):
+    todo = todo_factory(summary='harhar\n', priority=1)
+
+    result = runner.invoke(cli, ['show', '1'])
+    assert '1' in result.output
```

Comment:
Now, all test cases are passed except the one ```run_test(sort_key)```. I don't understand why it is failed.